webapp/storybook/config.js

```diff
@@ -22,6 +22,20 @@ Vue.component('v-popover', {
   template: '<div><slot>Popover Content</slot></div>',
 })
 
+// Setup design token addon
+const scssReq = require.context('!!raw-loader!~/view/styles', true, /.\.scss$/)
+const scssTokenFiles = scssReq
+  .keys()
+  .map(filename => ({ filename, content: scssReq(filename).default }))
```

Comment:
That's quite interesting. I wonder in which places we could re-use that. Did you copy this from somehwere @alina-beck?

Reply:
Yep, this came from the [design token addon](https://github.com/UX-and-I/storybook-design-token) for storybook.

I don't think it's possible to change token values _in_ the storybook interface but you should be able to change them in the `.scss` file and see the change reflected in storybook immediately.